README.md

```diff
@@ -24,10 +24,24 @@ The process is divided in five steps:
 2. start by editing your colorscheme's information,
 3. define your colors,
 4. define your highlight groups and links,
-5. and generate your colorscheme.
+5. and generate your colorscheme (refer next step below).
 
 Steps 2 to 5 are thoroughly described in the colorscheme template itself in an effort to make it portable: if you ever decide to distribute your colorscheme you can simply package the template with it.
 
+## How to Generate?
+
+you need to have ruby installed `sudo apt install ruby` before get started.
```

Comment:
This is too OS-specific. We should either explain it for every OS or refer to [some official document](https://www.ruby-lang.org/en/documentation/installation/).